CHANGELOG.md

```diff
@@ -19,6 +19,7 @@
     * Support for signing in a group transaction (loaded from file).
 + Add docs for web package
 + Handle group transaction in web package
++ Deploy ASA without using asa.yaml
```

Comment:
```suggestion
+ Deploy ASA without using `/assets/asa.yaml`
```

Reply:
Could you please add a link to docs as well [https://github.com/scale-it/algo-builder/pull/434/files#diff-589c6e32b83e24109f0b5631b5a08a948bf336c1e007f3f473127af7e825668aR35](https://github.com/scale-it/algo-builder/pull/434/files#diff-589c6e32b83e24109f0b5631b5a08a948bf336c1e007f3f473127af7e825668aR35)